test/shared/Id.js

```diff
@@ -12,6 +12,7 @@ module.exports = function Id(value) {
     'fantasy-land/chain': function(f) { return f(value); },
     'fantasy-land/reduce': function(f, x) { return f(x, value); },
     'fantasy-land/traverse': function(f, of) { return R.map(Id, f(value)); },
+    'fantasy-land/promap': function(f, g) { return Id(R.pipe(f, value, g)); },
```

Comment:
`Id a` cannot satisfy the requirements of Profunctor for the same reasons that `Maybe a` cannot.